### DISCUSSION TITLE
Oct. 8 Team Check-in

### DISCUSSION THREAD

**Discussion by Wes-Mac** [ecn310/course-project-zipcentercrime#9] [General]
# For Each Member: #
- Something someone did to help the team this week:
- Something I could have done better for the team this week:
- An area / suggestion for team improvement:

**Comment by sortizh** [comment]
## Sophia 10/8/24
- *Something someone did to help the team this week:* Rachel found articles on average distance traveled so we can use that for outlier reasoning 
- *Something I could have done better for the team this week:* I could have directed people's tasks better as my role as a mediator, and made sure people's messages in the group chat didn't go unanswered
- *An area / suggestion for team improvement:* State outloud what you are doing during meetups so everyone knows and can better coordinate with each other

**Comment by Wes-Mac** [comment]
# Weston 10/08/24

- Rachel organized a time for our group to meet with professor Deza which helped us under stand the project as well as get the remaining data we needed
- I could have put more effort into finding out the deadlines for certain projects as the timekeeper of the group
- We should think about starting a group calendar and adding to is as we learn about future deadlines and prevent times where we don't have things done by the time they need to be

**Comment by Gersh305** [comment]
- **Something someone did to help the team this week:** The rest of the team had a very productive meeting with professor Deza that helped to clarify a lot of information we needed for our project
- **Something I could have done better for the team this week:** I could have found my papers faster and done more for their lit review
- **An area / suggestion for team improvement:** We are constantly late for our team's weekly meeting so it might be better to modify the time. We said we'd meet at 5, but we consistently start working around 5:20-5:30. It isn't fair to Wes that he has to get there early

**Comment by regaudre** [comment]
### Rachel Gaudreau 10/8/24
- Something someone did to help the team this week: Sophia helped us manage tasks and keep on track with deadlines.
- Something I could have done better for the team this week: Keep more on top of information, and making sure I am posting information to GitHub in a timely manner.
- An area / suggestion for team improvement: Have a better plan with what we need to accomplish as a group as a whole.
